crates/api_models/src/enums.rs

```diff
@@ -508,20 +508,25 @@ pub struct UnresolvedResponseReason {
 #[serde(rename_all = "snake_case")]
 #[strum(serialize_all = "snake_case")]
 pub enum FieldType {
+    CardNumber,
+    CardExpiryMonth,
+    CardExpiryYear,
+    CardCVC,
```

Comment:
Nit (naming convention):

```suggestion
    CardCvc,
```

Can be taken up in a different PR.